file-formats/ints/type/zif_aff_ints_v1.intf.abap

```diff
@@ -0,0 +1,390 @@
+      "! <p class="shorttext">Additional information position ID</p>
+      "! Additional information position ID
+      position_id         TYPE i,
+    END OF ty_islm_psaddlinfo,
+
+    "! <p class="shorttext">Scenario additional information texts</p>
+    "! Scenario additional information texts
+    BEGIN OF ty_islm_psaddlinfot,
+      "! <p class="shorttext">Language Key</p>
+      "! Language Key
+      language            TYPE sy-langu,
+      "! <p class="shorttext">Additional information key</p>
+      "! Additional information key
+      additional_info_key TYPE c LENGTH 50,
+      "! <p class="shorttext">Additional information value ID</p>
+      "! Additional information value ID
+      value_id            TYPE i,
+      "! <p class="shorttext">Additional information key text label</p>
+      "! Additional information key text label
+      key_label           TYPE c LENGTH 255,
+      "! <p class="shorttext">Additional information value text label</p>
+      "! Additional information value text label
+      value_label         TYPE c LENGTH 255,
+    END OF ty_islm_psaddlinfot,
+
+    "! <p class="shorttext">SBS Scenario Segment Fields</p>
+    "! SBS Scenario Segment Fields
+    BEGIN OF ty_islm_pssgmntfld,
+      "! <p class="shorttext">Table or view field name</p>
+      "! Table or view field name
+      fieldname      TYPE c LENGTH 30,
+      "! <p class="shorttext">Position</p>
+      "! Position
+      field_position TYPE i,
+      "! <p class="shorttext">Key field or not</p>
+      "! Key field or not
+      iskey          TYPE c LENGTH 1,
+    END OF ty_islm_pssgmntfld,
+
+    "! <p class="shorttext">Intelligent Scenario Models</p>
+    "! Intelligent Scenario Models
+    BEGIN OF ty_models,
+      "! <p class="shorttext">Model ID</p>
+      "! Model ID
+      guid              TYPE c LENGTH 36,
+      "! <p class="shorttext">Model Name</p>
+      "! Model Name
+      "! $required
+      model_name        TYPE c LENGTH 30,
+      "! <p class="shorttext">Model Description</p>
+      "! Model Description
+      description       TYPE string,
+      "! <p class="shorttext">State of Model</p>
+      "! State of Model
+      state             TYPE c LENGTH 10,
+      "! <p class="shorttext">Executable ID</p>
+      "! Executable ID
+      "! $required
+      executable_id     TYPE c LENGTH 64,
+      "! <p class="shorttext">Large Language Model Name</p>
+      "! Large Language Model Name
+      "! $required
+      llm_name          TYPE string,
+      "! <p class="shorttext">LLM Model Version</p>
+      "! LLM Model Version
+      "! $required
+      llm_model_version TYPE c LENGTH 30,
+    END OF ty_models,
+
+    "! <p class="shorttext">Model Prompt Template</p>
+    "! Model Prompt Template
+    BEGIN OF ty_islm_mdlprmttmpl,
+      "! <p class="shorttext">Model Name</p>
+      "! Model Name
+      "! $required
+      model_name        TYPE c LENGTH 30,
+      "! <p class="shorttext">Prompt template name</p>
```

Comment:
Words in the title must be capitalised, except for connectors: https://github.com/SAP/abap-file-formats/blob/main/docs/json.md#title

Reply:
Please check all titles